### PR TITLE
Made notifications more verbose.

### PR DESCRIPTION
**Pull request by saleem-latif**

### CODE REVIEW

File: src/main/java/org/intellij/sdk/codesync/actions/FilePlaybackAction.java [hunk 1]
```diff
@@ -78,11 +81,15 @@ public void actionPerformed(@NotNull AnActionEvent e) {
 
         if (configRepo == null) {
             NotificationManager.notifyError(
-                    "An error occurred trying to perform file playback action.", project
+                String.format(
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/actions/FilePlaybackAction.java:84]: It think If repo is not synced, View file playback Or repo playback options should not be visible

**Review by Basit-qc — APPROVED** [review]
Looks good

File: src/main/java/org/intellij/sdk/codesync/actions/FilePlaybackAction.java (hunk 1 above)
Comment (saleem-latif) [reply src/main/java/org/intellij/sdk/codesync/actions/FilePlaybackAction.java:84]: Yes, that is the case as of now as well. This line here is just for completeness. Ideally this should never happen.

ref: https://github.com/codesyncapp/codesync-intellij/blob/master/src/main/java/org/intellij/sdk/codesync/actions/RepoPlaybackAction.java#L25

Comment (Basit-qc) [reply src/main/java/org/intellij/sdk/codesync/actions/FilePlaybackAction.java:84]: Sounds good. Thanks

**Pull request merged**